patch.js

```diff
@@ -184,7 +228,8 @@ function forgeCertificateChain(options) {
 
 	// * Update the CN and resign
 	csr.publicKey = sitePublicKey;
-	csr.setSubject([ // TODO - Add the ability to set more of these?
+	csr.setSubject([
+		// TODO - Add the ability to set more of these?
```

Comment:
This and the other one were automatic formatter changes. Is it specifically a problem?

Reply:
Not a problem per-say, I just don't particulalry like "changes for the sake of changes" types of changes. I know in this case it was an automatic formatter change, but it's still a change that brings no actual value (just not done intentionally). It clutters the diff in my opinion and doesn't add anything to the pull request (also I have to be fair to everyone, if I allow some "changes for the sake of changes" for one person then I have to do it for everyone, and that opens to door to people making useless PRs like this https://github.com/PretendoNetwork/website/pull/267)

Reply:
I understand, but I think there is a fair difference between making a useless PR and making a useless change to a file that is already being edited.

Reply:
I disagree, since as I said it still clutters the diff for no added value. I discourage them for various reasons, including both useless PRs and cluttering the diff

Reply:
I understand your opinion on this. Does this change need to be reverted?

Reply:
No this time is fine, but we should probably set some standards when it comes to auto formatters. I've mentioned this in this task as well https://github.com/orgs/PretendoNetwork/projects/34/views/1?pane=issue&itemId=65829524